converter/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java

```diff
@@ -101,7 +101,11 @@
 	CPC_PLUS_TOO_FEW_QUALITY_MEASURE_CATEGORY(64,
 			"CPC+ Submissions must have at least %d of the following %s measures: %s.", true),
 	CPC_PLUS_TOO_FEW_QUALITY_MEASURES(65,
-			"CPC+ Submissions must have at least %d of the following measures: %s.", true);
+			"CPC+ Submissions must have at least %d of the following measures: %s.", true),
+	CPC_PLUS_MISSING_SUPPLEMENTAL_CODE(66,
```

Comment:
Be careful with this merge because I believe error code 66 already exists in the master branch